auth_api_key/models/auth_api_key.py

```diff
@@ -13,10 +14,14 @@ class AuthApiKey(models.Model):
     _inherit = "server.env.mixin"
     _description = "API Key"
 
+    def _default_key(self):
+        return uuid.uuid4()
+
     name = fields.Char(required=True)
     key = fields.Char(
         required=True,
-        help="""The API key. Enter a dummy value in this field if it is
+        default=_default_key,
```

Comment:
@danypr92 To allow an easy override of the default method, you should avoid to make a direct link to the method into the field definition. `default=lambda self: self._default_key()`

Reply:
Thanks @lmignon !!

I push a commit with your suggestion. :rocket: